environments/barista-environment.ts

```diff
@@ -14,17 +14,22 @@
  * limitations under the License.
  */
 
-import { join } from 'path';
 import { isPublicBuild } from '@dynatrace/tools/shared';
+import { config as dotenvConfig } from 'dotenv';
+import { join } from 'path';
 import { BaEnvironment } from './barista-environment.interface';
 
+// TODO: Remove all platform dependent usages like process and dotenv
+// load the environment variables from the .env file in your workspace
```

Comment:
I am confused :) should we use dotenv and a .env file now or not?

Reply:
this is not a file consumed by angular :) only from nodejs. but i don't like it to access the process.env but we do it already so I added a todo to remove it later on I prefer the approach to add a barista-environment.internal.ts that is generated by our jenkins. But this should be part of a different issue

Reply:
I agree with @lukasholzer here, we should try and move everything that was configured in the process.env to the .env file. 
We just need to make sure that the .env file is not within the git index.